napari/components/viewer_model.py

```diff
@@ -704,6 +704,28 @@ def _calc_status_from_cursor(
         selection = self.layers.selection
         active = selection.active
 
+        # the tooltip is computed in all cases, so we compute it first
```

Comment:
```suggestion
        # Compute the tooltip first since it is always needed.
```

All cases was confusing to me.